Dockerfile.v3.2.x

```diff
@@ -40,6 +40,6 @@ COPY entrypoint.sh /entrypoint.sh
 WORKDIR /opt/tplink/EAPController
 EXPOSE 8088 8043 27001/udp 27002 29810/udp 29811 29812 29813
 HEALTHCHECK --start-period=5m CMD wget --quiet --tries=1 --no-check-certificate -O /dev/null --server-response --timeout=5 https://127.0.0.1:8043/login || exit 1
-VOLUME ["/opt/tplink/EAPController/data","/opt/tplink/EAPController/work","/opt/tplink/EAPController/logs"]
+VOLUME ["/opt/tplink/EAPController/data","/opt/tplink/EAPController/work","/opt/tplink/EAPController/logs","/cert"]
```

Comment:
I would skip including `/cert` in the `VOLUME` line because it isn't necessary from a persistent data standpoint and will create a docker named volume for other users unless you know of a good reason why it should be included.

Reply:
Yes, you are right /certs can be overvwritten also without mentioning it as volume.